lib/ex_twilio/capability.ex

```diff
@@ -253,10 +254,7 @@ defmodule ExTwilio.Capability do
   end
 
   defp generate_jwt(payload, secret) do
-    payload
-    |> Joken.token()
-    |> Joken.with_signer(Joken.hs256(secret))
-    |> Joken.sign()
-    |> Joken.get_compact()
+    signer = Joken.Signer.create("HS256", secret || "")
```

Comment:
I think we might want to raise an error if `secret` is nil?

```suggestion
    signer = Joken.Signer.create("HS256", secret)
```

Reply:
i don't actually think the default value is necessary